WordPress/Classes/ViewRelated/Blog/Site Picker/SiteSwitcherView.swift

```diff
@@ -66,12 +66,32 @@ private struct SiteSwitcherToolbarView: View {
         if !isSearching {
             HStack {
                 Spacer()
-                FAB(action: addSiteAction)
+                button
                     .padding(.trailing, 20)
                     .padding(.bottom, UIDevice.current.userInterfaceIdiom == .pad ? 20 : 0)
             }
         }
     }
+
+    @ViewBuilder
+    private var button: some View {
+        let viewModel = AddSiteAlertViewModel(onSelection: addSiteAction)
+        switch viewModel.actions.count {
+        case 0:
+            EmptyView()
+        case 1:
+            FAB(action: viewModel.actions[0].handler)
+        default:
+            Menu {
+                // Menu reverses actions by default
+                ForEach(viewModel.actions.reversed()) { action in
+                    Button(action.title, action: action.handler)
+                }
+            } label: {
+                FAB(action: {})
```

Comment:
I presume you want the content, but not the action button in `FAB`? It'd be great if we can make the content view available for use. I noticed there seems to be some complications around dynamic type. Maybe that's a blocker in publishing the `FABContenxtView`?

Reply:
Oh, I added a variant that doesn't require an action, but forgot to use it – updated.

> I noticed there seems to be some complications around dynamic type. Maybe that's a blocker in publishing the FABContentView?

It's a similar situation as with `@Environment(isSearching`. `@ScaledMetric` has to be _inside_ the view that sets `dynamicTypeSize`. So there has a to be a wrapper for `FABContentView`. I think using `FAB` for this purpose is OK.